cpp/ql/test/query-tests/AlertSuppression/tst.c

```diff
@@ -9,7 +9,16 @@ int x = 0; // lgtm
 // lgtm blah blah #falsepositive
 //lgtm  [js/invocation-of-non-function]
 /* lgtm */
+/* lgtm[] */
```

Comment:
We generally try to avoid inserting tests in the middle of the file, since that causes any results in later lines to be renumbered, which inflates the diff of the `.expected` file.